### PR TITLE
feat(docs): document `discord` package

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
+//     |> last()
+//     |> findRecord(fn: (key) => true, idx: 0)
+//
+// discord.send(
+//   webhookToken:token,
+//   webhookID: "1234567890",
+//   username: "chobbs",
+//   content: "The current status is \"${lastReported.status}\".",
+//   avatar_url: "https://staff-photos.net/pic.jpg"
+// )
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:45]:
```suggestion
// import "contrib/chobbs/discord"
// import "influxdata/influxdb/secrets"
// 
// token = secrets.get(key: "DISCORD_TOKEN")
// 
// lastReported = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses")
//     |> last()
//     |> findRecord(fn: (key) => true, idx: 0)
// 
// discord.send(
//     webhookToken: token,
//     webhookID: "1234567890",
//     username: "chobbs",
//     content: "The current status is \"${lastReported.status}\".",
//     avatar_url: "https://staff-photos.net/pic.jpg",
// )
```

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 2]
```diff
@@ -1,16 +1,49 @@
...
+// ```
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:46]:
```suggestion
// ```
//
// tags: single notification
//
```

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 3]
```diff
@@ -25,13 +58,53 @@ send = (
...
-// `endpoint` creates a factory function that creates a target function for pipeline `|>` to send messages to discord for each table row.
-// `webhookToken` - string - the secure token of the webhook.
-// `webhookID` - string - the ID of the webhook.
-// `username` - string - username posting the message.
-// `avatar_url` -  override the default avatar of the webhook.
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `content`, as defined in the `send` function arguments.
+// endpoint sends a single message to a Discord channel using a [Discord webhook] and data from table rows.
+//
+// [Discord webhook]: https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks&?page=3
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:63]:
```suggestion
// endpoint sends a single message to a Discord channel using a
// [Discord webhook](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks&?page=3)
// and data from table rows.
```

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 4]
```diff
@@ -25,13 +58,53 @@ send = (
...
+//
+// ## Parameters
+// - webhookToken: Discord [webhook token](https://discord.com/developers/docs/resources/webhook).
+// - webhookID: Discord [webhook ID](https://discord.com/developers/docs/resources/webhook).
+// - username: Override the Discord webhook’s default username.
+// - avatar_url: Override the Discord webhook’s default avatar.
+//
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:70]:
```suggestion
// - avatar_url: Override the Discord webhook’s default avatar.
// - tables: Input data. Default is piped-forward data (`<-`).
```

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 5]
```diff
@@ -25,13 +58,53 @@ send = (
...
+// crit_statuses = from(bucket: "example-bucket")
+//   |> range(start: -1m)
+//   |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
+//
+// crit_statuses
+//   |> endpoint(mapFn: (r) => ({
+//       content: "The status is critical!",
+//     })
+//   )()
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:106]:
```suggestion
// import "influxdata/influxdb/secrets"
// import "contrib/chobbs/discord"
// 
// discordToken = secrets.get(key: "DISCORD_TOKEN")
// endpoint = telegram.endpoint(
//     webhookToken: discordToken,
//     webhookID: "123456789",
//     username: "critBot",
// )
// 
// crit_statuses = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
// 
// crit_statuses
//     |> endpoint(
//         mapFn: (r) => ({
//             content: "The status is critical!",
//         }),
//     )()
```

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 6]
```diff
@@ -25,13 +58,53 @@ send = (
...
+// ```
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:107]:
```suggestion
// ```
//
// tags: notifcation endpoints,transformations
//
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: stdlib/contrib/chobbs/discord/discord.flux [hunk 7]
```diff
@@ -25,13 +60,57 @@ send = (
...
+// [Discord webhook](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks&?page=3)
+// and data from table rows.
+//
+// ## Parameters
+// - webhookToken: Discord [webhook token](https://discord.com/developers/docs/resources/webhook).
+// - webhookID: Discord [webhook ID](https://discord.com/developers/docs/resources/webhook).
+// - username: Override the Discord webhook’s default username.
+// - avatar_url: Override the Discord webhook’s default avatar.
+// - tables: Input data. Default is piped-forward data (`<-`).
+// ## Usage
```
Comment (sanderson) [inline stdlib/contrib/chobbs/discord/discord.flux:73]:
```suggestion
//
// ## Usage
```

**Review by sanderson — APPROVED** [review]
One nit, and it's gtg.

**Review by skartikey — APPROVED** [review]

**Pull request merged**